__tests__/components/feedback_bar_test.js

```diff
@@ -53,32 +53,32 @@ describe("Feedback bar", () => {
     });
   });
 
-  it("editing the what were you doing text field updates the action state", () => {
+  it("editing the what were you doing text area updates the bug state", () => {
```

Comment:
I think this is no longer the "what were you doing" text area. Maybe the title could be: "clicking to report a bug and editing the text area updates the bug state"